daemon/anonymous-statistics.sh.in

```diff
@@ -71,6 +71,23 @@ NETDATA_PREBUILT_DISTRO="${42}"
 
 [ -z "$NETDATA_REGISTRY_UNIQUE_ID" ] && NETDATA_REGISTRY_UNIQUE_ID="00000000-0000-0000-0000-000000000000"
 
+KERNEL_NAME="$(uname -s)"
+MD5_PATH="$(exec <&- 2>&-; which md5sum || command -v md5sum || type md5sum)"
```

Comment:
will this handle or what will happen if md5sum does not exist?

Reply:
think its answered here :) https://chat.openai.com/share/1489a07a-5d1d-47f0-b35c-c7ba4138f2ff

Reply:
I wouldn't expect many systems not have the md5sum installed, so we should be safe. If not found, a uuid will be created instead. We'll take @Ferroin opinion too :-)